src/main/java/mario/model/map/MapLoader.java

```diff
@@ -30,30 +33,40 @@ private MapLoader(){}
         } catch (IOException e) {
             e.printStackTrace();
         }
-
         return map;
-
     }
 
+    /**
+     * This method loads an empty map from a file.
+     * @return int[][]
+     */
     public static int[][] loadEmptyMap() {
-
         return loadMap(new File(MainApp.MEDIA_URL.concat("/maps/empty.csv")));
     }
 
-    private static int[] toIntArray(String[] array) {
 
+    /**
+     * This method is a helper method for converting a numerical string array
+     * to an integer array.
+     * @param array an array of numerical strings.
+     * @return int[]
+     */
+    private static int[] toIntArray(String[] array) {
```

Comment:
Wellicht een duidelijkere methode naam geven. Als ik de omschrijving erboven lees zou je misschien kunnen zeggen: convertStringToIntArray?

Reply:
Ik vind dat als je de hele methode signature bekijkt, dat de naam in combinatie met het feit dat de methode een String array als parameter heeft, dat dat duidelijk genoeg is.